src/test/java/org/opensearch/knn/index/codec/KNN920Codec/KNN920CodecTests.java

```diff
@@ -55,93 +38,31 @@ public void testBuildFromModelTemplate() throws InterruptedException, ExecutionE
     }
 
     public void testKnnVectorIndex() throws Exception {
-        final String fieldName = "test_vector";
-        final String field1Name = "my_vector";
         final MapperService mapperService = mock(MapperService.class);
         final KNNMethodContext knnMethodContext = new KNNMethodContext(
             KNNEngine.LUCENE,
             SpaceType.L2,
             new MethodComponentContext(METHOD_HNSW, Map.of(HNSW_ALGO_M, 16, HNSW_ALGO_EF_CONSTRUCTION, 256))
         );
         final KNNVectorFieldMapper.KNNVectorFieldType mappedFieldType1 = new KNNVectorFieldMapper.KNNVectorFieldType(
-            fieldName,
+            FIELD_NAME_ONE,
```

Comment:
Will there be a way for this class doesn't need to know about `FIELD_NAME_ONE` and `FIELD_NAME_TWO`? 
This code might reduce code duplication but it seems it does not achieve a separation of concerns. I had to look in to `testKnnVectorIndex` code and what it does and how `FIELD_NAME_ONE` and `FIELD_NAME_TWO` is used. Maybe, it could be because I am not familiar with KNN code base though.

Reply:
I see what you mean, I'm also not that happy with this change. let me try few things.

Reply:
I think I've got it, will push the commit soon